### PR TITLE
Use log4cp always on travis

### PR DESCRIPTION
**Pull request by Liorgal28**
Insert Log4cplus as default logging in Travis.

### CODE REVIEW

**Comment by salieri11** [comment]
> _No description provided._

Please add at least couple of words about this PR. Use WIP: instead of "mock PR" if you want to test something.

File: tests/apollo/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 add_test(NAME skvbc_basic_tests COMMAND sh -c
-        "python3 -m unittest test_skvbc 2>&1 > /dev/null"
+        "python3 -m unittest test_skvbc"
```
Comment (teoparvanov) [inline tests/apollo/CMakeLists.txt]: you can now drop this change

File: tests/apollo/test_skvbc.py [hunk 2]
```diff
@@ -65,6 +65,7 @@ class SkvbcTest(unittest.TestCase):
             bft_network.all_replicas(without={0, stale_node}))
         bft_network.stop_replica(random_replica)
         await skvbc.assert_successful_put_get(self)
+        assert True is False
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc.py]: same here - we don't need this anymore

**Comment by teoparvanov** [comment]
Hmm, the CI run seems to be stuck. Can you remove the "WIP" from the commit title and push, so that CI can rerun?

**Pull request merged**